src/py-opentimelineio/opentimelineio/adapters/cmx_3600.py

```diff
@@ -627,6 +627,17 @@ def parse(self, comment):
                 self.unhandled.append(stripped)
 
 
+def _get_next_clip(start_index, track):
+    """Get the next clip with a non-zero duration"""
+    # Iterate over the following clips and return the first "real" one
+    for index in range(start_index + 1, len(track)):
+        clip = track[index]
+        if clip.duration().value > 0:
+            return clip
```

Comment:
```suggestion
    for clip in track[start_index+1:]:
        if clip.duration().value > 0:
            return clip
```

I think this might simplify that loop a bit.